python/pycylon/pycylon/data/aggregates.pyx

```diff
@@ -36,3 +42,86 @@ AggregationOpString = {
     'quantile': CGroupByAggregationOp.CQUANTILE,
     'std': CGroupByAggregationOp.CSTDDEV,
 }
+
+def sum_table(ctx: CylonContext, table: Table, skipna=True) -> Column:
+    cdef shared_ptr[CCylonContext] cctx = pycylon_unwrap_context(ctx)
+    cdef shared_ptr[CTable] ctable = pycylon_unwrap_table(table)
+    cdef CBasicOptions options = CBasicOptions(skipna)
+    cdef shared_ptr[CColumn] cresult
+    cdef CStatus status = SumTable(cctx, ctable, &cresult, options)
+
+    if status.is_ok():
+        return pycylon_wrap_column(cresult)
+    else:
+        raise Exception(f"aggregation error: {status.get_msg().decode()}")
+
+def min_table(ctx: CylonContext, table: Table, skipna=True) -> Column:
+    cdef shared_ptr[CCylonContext] cctx = pycylon_unwrap_context(ctx)
+    cdef shared_ptr[CTable] ctable = pycylon_unwrap_table(table)
+    cdef CBasicOptions options = CBasicOptions(skipna)
+    cdef shared_ptr[CColumn] cresult
```

Comment:
nit: this seems like a separate cdef function which can be called within each of these aggregate functions.
WDYT?

Reply:
unless we use some sort of an enum, I'm not sure how we can achieve this.